instrumentation.node.ts

```diff
@@ -4,10 +4,16 @@ import {
 	noopSpanProcessor,
 } from "@/lib/opentelemetry";
 import { registerOTel } from "@vercel/otel";
+import { LangfuseExporter } from "langfuse-vercel";
 
 registerOTel({
 	serviceName: "giselle",
 	spanProcessors: [noopSpanProcessor],
 	metricReader,
 	logRecordProcessor,
+	traceExporter: new LangfuseExporter({
```

Comment:
I can't expect where the telemetry sent because this initialization does not contain target endpoint.
But reading the [Langfuse documant](https://langfuse.com/changelog/2024-08-02-vercel-ai-sdk-integration), this configuration seems ok.
Let's merge and check that new fields are included in the telemetry arrived to our Langfuse account

Reply:
I think having "staging" environment is good if we accept pull requests from forked repo: we can check new functionalities before merging them into `main`

Reply:
@Rindrics  It deployed to staging. please verify!